x/auth/mock/simulate_block.go

```diff
@@ -150,12 +139,13 @@ func SignDeliver(t *testing.T, app *baseapp.BaseApp, msg sdk.Msg, seq []int64, e
 	app.EndBlock(abci.RequestEndBlock{})
 }
 
-func GetAllAccounts(app *App, ctx sdk.Context) []auth.Account {
+// Get all accounts in the accountMapper
+func GetAllAccounts(mapper auth.AccountMapper, ctx sdk.Context) []auth.Account {
```

Comment:
Should this be used in places outside of testing? e.g.
```
examples/basecoin/app/app.go
172:	app.accountMapper.IterateAccounts(ctx, appendAccount)

examples/democoin/app/app.go
171:	app.accountMapper.IterateAccounts(ctx, appendAccount)
```
Not sure if we should import this helper outside of test files.